packages/framework/presence/src/presenceDatastoreManager.ts

```diff
@@ -402,15 +436,13 @@ export class PresenceDatastoreManagerImpl implements PresenceDatastoreManager {
 				let workspaceDatastore = this.datastore[workspaceAddress];
 				if (workspaceDatastore === undefined) {
 					workspaceDatastore = this.datastore[workspaceAddress] = {};
-					if (!workspaceAddress.startsWith("system:")) {
-						// TODO: Emit workspaceActivated event for PresenceEvents
-					}
 				}
```

Comment:
I think this can now be simplified to
```suggestion
				// Ensure there is a datastore at this address and get it.
				const workspaceDatastore = (this.datastore[workspaceAddress] ??= {});
```